test/MusicStore.Test/ShoppingCartControllerTest.cs

```diff
@@ -27,6 +30,8 @@ public ShoppingCartControllerTest()
                       .AddInMemoryStore()
                       .AddDbContext<MusicStoreContext>();
 
+            services.AddMvc();
```

Comment:
`AddMvc` looks lot simpler and familiar than adding these many lines which requires to know about service types.